### PR TITLE
minimal install compatibility

### LINKED ISSUE #158: role attempts to remediate SSH even if not installed
If openssh-server is not installed, none of the SSH remediations need to be performed.  Currently, the role fails out in this case.

### LINKED ISSUE #160: disable wifi fails without NetworkManager installed
PR incoming.

### PR DESCRIPTION
**Pull request by jamescassell**
These fixes improve compatibility with a minimal install.

fixes #158
fixes #160

### CODE REVIEW

File: tasks/prelim.yml [hunk 1]
```diff
@@ -56,6 +56,19 @@
   tags:
       - always
 
+- name: ensure cronie is available
+  yum:
+      name: cronie
+  when:
+      - rhel_07_020030 or
+        rhel_07_020040 or
+        false
```
Comment (shepdelacreme) [inline tasks/prelim.yml]: What is the purpose of the `or false`?

**Comment by shepdelacreme** [comment]
LGTM. Just one comment.

File: tasks/prelim.yml (hunk 1 above)
Comment (jamescassell) [reply tasks/prelim.yml]: Allows reordering the above items without worrying about which one doesn't have an or at the end, and makes it so adding new items doesn't require adding or to an existing item

Comment (jamescassell) [reply tasks/prelim.yml]: But in this case, the list is quite short and unlikely to change frequently. I'm happy to remove it if that is preferred.

Comment (shepdelacreme) [reply tasks/prelim.yml]: I'm fine leaving it in I just wasn't sure what the purpose was lol. Now it makes sense.

**Pull request merged**